api-spec/openapi/STAC.yaml

```diff
@@ -328,6 +319,21 @@ components:
           example: 'http://www.geoserver.example/stac/naip/child/catalog.json'
         rel:
           example: child
+        method:
+          type: string
+          enum:
+            - GET
```

Comment:
`get` or `GET`?

Reply:
Following RFC7231 we should have it uppercase.
The RFC states in section 4.1: 
> By convention, standardized methods are defined in all-uppercase US-ASCII letters.

Reply:
But the default value should be uppercase, too.

Reply:
Good catch, fixed.